lib/model/localizations.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter_gen/gen_l10n/zulip_localizations.dart';
+import '/generated/l10n/zulip_localizations.dart';
```

Comment:
Should we use relative path like we do in the other files, or is it needed for other reasons?